src/classes/modules/java.xml/jdk/xml/internal/SecuritySupport.java

```diff
@@ -0,0 +1,201 @@
+package jdk.xml.internal;
```

Comment:
Please add the copyright/license header here.

Reply:
Done Cyrille. Thanks for your patience!